app/src/main/java/org/jellyfin/androidtv/util/apiclient/ApiClientExtensions.kt

```diff
@@ -55,12 +55,14 @@ suspend fun ApiClient.updateFavoriteStatus(itemId: String, userId: String, isFav
 	})
 }
 
-suspend fun ApiClient.getSimilarItems(item: BaseItem): List<BaseItem>? = suspendCoroutine {continuation ->
-	val query = SimilarItemsQuery()
-	query.setUserId(TvApp.getApplication().currentUser.id)
-	query.setId(item.id)
-	query.setLimit(10)
-	query.fields = FieldsRequiredForLift
+suspend fun ApiClient.getSimilarItems(item: BaseItem, limit: Int = 25): List<BaseItem>? = suspendCoroutine { continuation ->
+	val query = SimilarItemsQuery().apply {
+		id = item.id
+		userId = TvApp.getApplication().currentUser.id
+		this.limit = limit
```

Comment:
I _think_ this sets the value of query.limit to query.limit

Reply:
IntelliJ syntax highlighting highlighted the other limit, I'll check with the debugger, just to be sure

Reply:
Works